package.json

```diff
@@ -5,12 +5,11 @@
   "main": "dist/cms.js",
   "scripts": {
     "start": "npm run dev",
-    "dev": "webpack-serve webpack.dev.js",
-    "dev:write": "webpack-serve webpack.dev-write.js",
+    "dev": "webpack-dev-server --env.development",
+    "dev:write": "webpack-dev-server --env.development --env.write",
     "test": "jest --coverage",
     "test:watch": "jest --watch",
-    "build": "cross-env NODE_ENV=production webpack-cli --config webpack.prod.js --display-error-details",
-    "build:scripts": "cross-env NODE_ENV=production webpack-cli --config webpack.cli.js",
+    "build": "cross-env NODE_ENV=production webpack-cli --display-error-details --env.production",
```

Comment:
Is there any problem with removing the `NODE_ENV` value, since we are not using it in the script? The `mode` sets it in the actual bundle.

Reply:
The problem here is the bundle size without it, I am wondering if the mode is not setting it on windows? needs research.

Reply:
Looks like it's not getting passed through to Babel. Either way, we might as well leave it, not a big deal.